spec/models/list_item_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+RSpec.describe ListItem do
+  describe "#display_title" do
+    it "shows the rummager title by default" do
+      stub_request(:get, RummagerHelper::SEARCH_ENDPOINT)
```

Comment:
We could extract the method into a module. https://github.com/alphagov/collections-publisher/blob/fix-curated-list-item-titles/spec/support/rummager_helper.rb would be the place for that probably.